frontend/src/metabase/questions/containers/EntityList.jsx

```diff
@@ -57,45 +58,45 @@ const mapDispatchToProps = {
 const SECTIONS = [
     {
         id: 'all',
-        name: 'All questions',
+        name: t`All questions`,
         icon: 'all',
-        empty: 'No questions have been saved yet.',
+        empty: t`No questions have been saved yet.`,
     },
     {
         id: 'fav',
-        name: 'Favorites',
+        name: t`Favorites`,
         icon: 'star',
-        empty: 'You haven\'t favorited any questions yet.',
+        empty: t`You haven\'t favorited any questions yet.`,
     },
     {
         id: 'recent',
-        name: 'Recently viewed',
+        name: t`Recently viewed`,
         icon: 'recents',
-        empty: 'You haven\'t viewed any questions recently.',
+        empty: t`You haven\'t viewed any questions recently.`,
     },
     {
         id: 'mine',
-        name: 'Saved by me',
+        name: t`Saved by me`,
         icon: 'mine',
-        empty:  'You haven\'t saved any questions yet.'
+        empty:  t`You haven\'t saved any questions yet.`
     },
     {
         id: 'popular',
-        name: 'Most popular',
+        name: t`Most popular`,
         icon: 'popular',
-        empty: 'The most viewed questions across your company will show up here.',
+        empty: t`The most-viewed questions across your company will show up here.`,
     },
     {
         id: 'archived',
-        name: "Archive",
+        name: t`Archive`,
         icon: 'archive',
-        empty: 'If you no longer need a question, you can archive it.'
+        empty: t`If you no longer need a question, you can archive it.`
     }
 ];
 
 const DEFAULT_SECTION = {
     icon: 'all',
-    empty: 'There aren\'t any questions matching that criteria.'
+    empty: t`There aren\'t any questions matching that criteria.`
```

Comment:
You don't need this back slash anymore now that this is converted to use the template literal.

Reply:
Gotcha, that makes sense.